ep/build-velox/src/build_velox.sh

```diff
@@ -99,22 +117,42 @@ function compile {
   COMPILE_OPTION="$COMPILE_OPTION -DCMAKE_BUILD_TYPE=${BUILD_TYPE}"
   COMPILE_TYPE=$(if [[ "$BUILD_TYPE" == "debug" ]] || [[ "$BUILD_TYPE" == "Debug" ]]; then echo 'debug'; else echo 'release'; fi)
   echo "COMPILE_OPTION: "$COMPILE_OPTION
-  make $COMPILE_TYPE EXTRA_CMAKE_FLAGS="${COMPILE_OPTION}"
+
+  if [ $ARCH == 'x86_64' ]; then
+    make $COMPILE_TYPE EXTRA_CMAKE_FLAGS="${COMPILE_OPTION}"
+  elif [ $ARCH == 'arm64' ]; then
+    CPU_TARGET="arm64" make $COMPILE_TYPE EXTRA_CMAKE_FLAGS="${COMPILE_OPTION}"
```

Comment:
nit: not sure here may add a line breaker or use `cmdA && cmdB`?

Reply:
```
➜  ~ aa="bb" cat in.txt
hello
world
;;;;;%
➜  ~ echo $aa

➜  ~ aa="bb" && cat in.txt
hello
world
;;;;;%
➜  ~ echo $aa
bb
➜  ~
```

Reply:
I follow the meta/velox docs. see https://github.com/facebookincubator/velox/#setting-up-on-macos